src/main/java/service/Ec2Service.java

```diff
@@ -1,48 +1,147 @@
 package service;
 
+import static constants.ComponentConstant.BILLING;
+import static constants.ComponentConstant.BOM_COMPONENTS;
+import static constants.ComponentConstant.CORE_BOM;
+import static constants.ComponentConstant.FULFILLMENT;
+import static constants.ComponentConstant.ORDER_MANAGEMENT;
+import static constants.ComponentConstant.REVENUE_RECOGNITION;
+import static constants.ComponentConstant.TEST_UTILITIES;
+
 import com.amazonaws.regions.Regions;
-import com.amazonaws.services.ec2.AmazonEC2;
 import com.amazonaws.services.ec2.AmazonEC2ClientBuilder;
 import com.amazonaws.services.ec2.model.*;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import model.ServiceStatus;
 
 public class Ec2Service {
 
   private static final Regions DEFAULT_REGION = Regions.EU_WEST_1;
+  private static final Filter RUNNING_INSTANCE_FILTER = new Filter(
+      "instance-state-name").withValues("running");
+  private final TagService tagService = new TagService();
+  private final FilterService filterService = new FilterService();
+
+  private static final HashMap<String, List<ServiceStatus>> COMPONENT_SERVICES_STATUS_MAP = new HashMap<>(
+      Map.of(
+          BILLING, Collections.emptyList(),
+          ORDER_MANAGEMENT, Collections.emptyList(),
+          FULFILLMENT, Collections.emptyList(),
+          REVENUE_RECOGNITION, Collections.emptyList(),
+          CORE_BOM, Collections.emptyList(),
+          TEST_UTILITIES, Collections.emptyList()
+      ));
+
 
-  public void listInstance() {
-    AmazonEC2 ec2 = AmazonEC2ClientBuilder.standard().withRegion(DEFAULT_REGION).build();
+  public void listRunningInstance() {
```

Comment:
This method is quite large and doing more than one thing, maybe some of it could be split out further in to some private methods? E.g. the printing at the end